tests/test_detect_target.py

```diff
@@ -50,12 +48,37 @@ def image_zidane():
     assert zidane_image is not None
     yield zidane_image
 
+def rmse(actual: np.ndarray,
```

Comment:
Nit: 2 lintes between top level functions and classes.

Reply:
What I meant is that there are also 2 lines in between top level functions as well.